.github/workflows/vulnerability-scanning-on-repo.yml

```diff
@@ -11,15 +11,19 @@ on:
         required: false
         default: .
         type: string
+      package-directory:
+        required: false
+        default: .
+        type: string
 
 env:
-  BOM_FILE_PATH: "${{ inputs.working-directory }}/${{ inputs.APP_NAME }}.sbom.json"
+  BOM_FILE_NAME: "${{ inputs.APP_NAME }}${{ inputs.package-directory != '.' && inputs.package-directory || '' }}.sbom.json"
```

Comment:
I assume this actually works as a way to conditionally generate a value?

Reply:
[publish_frontend](https://github.com/telicent-oss/telicent-access/blob/main/.github/workflows/publish.yml#L80-L92) has package-directory set
```yml
  publish_frontend:
    uses: telicent-oss/shared-workflows/.github/workflows/docker-push-to-registries.yml@TELFE-211/scan-secrets
    with:
      APP_NAME: access
      package-directory: frontend
      # ....
```

[publish_api](https://github.com/telicent-oss/telicent-access/blob/main/.github/workflows/publish.yml#L94-L105) does not
```yml
  publish_frontend:
    uses: telicent-oss/shared-workflows/.github/workflows/docker-push-to-registries.yml@TELFE-211/scan-secrets
    with:
      APP_NAME: access
      package-directory: frontend
      # ....
```
Resulting in:
<img width="257" alt="Screenshot 2024-07-30 at 15 07 59" src="https://github.com/user-attachments/assets/1d824b49-d53e-42f2-b59c-484600a51615">

https://github.com/telicent-oss/telicent-access/releases/tag/v1.0.1

An obvious fast-follow improvement would be a delimiter.

Reply:
Added to Clean up epic https://telicent.atlassian.net/browse/TELFE-515